packages/flame/lib/src/experimental/expanded_component.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flame/components.dart';
+import 'package:flame/experimental.dart';
+
+/// Works similarly to flutter's Expanded widget.
+/// This component must be a direct child of a [LinearLayoutComponent].
```

Comment:
Add an assertion about this in `onMount`

Reply:
Won't an exception caused by the `ParentIsA` mixin happen in roughly the same time as an assertion in `onMount`?

Reply:
True, missed that mixin!